app/models/restaurant.rb

```diff
@@ -7,23 +7,20 @@
 #  rating     :decimal(, )      default(0.0)
 #  tenbis     :boolean
 #  address    :string
-#  price      :decimal(, )
 #  maxDevTime :decimal(, )
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
 #  cuisine_id :integer
+#  kosher     :boolean
 #
 
 class Restaurant < ApplicationRecord
   has_many :reviews , dependent: :destroy
   has_many :users , :through => :reviews
   belongs_to :cuisine
 
-
-  #validates :name , :address ,presence:true,allow_nil:false, allow_blank: true
   validates :name, :presence => true, :allow_nil => false, :allow_blank => false
-  validates :price, numericality: true
-
+  validates :name, :uniqueness => true
```

Comment:
Hi Itai!

Checkout Digi's style guide, these should be in Ruby's syntax of `uniqueness: true`

https://github.com/WeConnect/style-guide/tree/master/ruby#hash-syntax